docs/src/components/StandardHTMLAttributes.mdx

```diff
@@ -0,0 +1,9 @@
+import { Link } from '@aws-amplify/ui-react';
+import { Fragment } from '@/components/Fragment';
+
+### Standard HTML attributes
+
+The <code>{props.component}</code> will accept any of the standard HTML attributes that a <code>{props.element}</code> element accepts.
+Standard <code>{props.element}</code> attributes can be found at <Link isExternal="true" href={`https://developer.mozilla.org/en-US/docs/Web/HTML/Element/${props.link||props.component}#attributes`}>{props.link||props.component}</Link>
```

Comment:
For the link's text, I'm wondering if we can say something like "...can be found at the MDN Docs." That might keep it simple and avoid small inconsistencies like capitalization (e.g., `Button` on the Button page and `button` on the ToggleButton page)